### PR TITLE
feat(translation): french translation

### PR DESCRIPTION
**Pull request by cateland**
adding french translation

### CODE REVIEW

File: lang/fr.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  "WEAPON.NAME": "Nom",
+  "WEAPON.CATEGORY": "Type",
+  "WEAPON.GRIP": "Prise",
+  "WEAPON.BONUS": "Bonus",
+  "WEAPON.DAMAGE": "Dégâts",
+  "WEAPON.RANGE": "Portée",
+  "WEAPON.FEATURE": "Fonctionnalité",
+  "WEAPON.DESCRIPTION": "Description",
+
+  "SPELL.NAME": "nom",
```
Comment (Perfectro) [inline lang/fr.json:92]: Missing an uppercase here.

File: lang/fr.json [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  "SPELL.RANGE": "Portée",
+  "SPELL.DURATION": "Durée",
+  "SPELL.INGREDIENT": "Ingrédient",
+  "SPELL.DESCRIPTION": "Description",
+
+  "CRITICAL_INJURY.NAME": "Nom",
+  "CRITICAL_INJURY.LETHAL": "Létal",
+  "CRITICAL_INJURY.LIMIT": "Limite de temps",
+  "CRITICAL_INJURY.EFFECT": "Effets",
+  "CRITICAL_INJURY.HEALING_TIME": "Temp de guérison",
```
Comment (Perfectro) [inline lang/fr.json:103]: Missing an 's' to `Temp`

File: lang/fr.json [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  "ARTIFACT.DESCRIPTION": "Description",
+  "ARTIFACT.DRAWBACK": "Inconvénient",
+  "ARTIFACT.APPEARANCE": "Apparence",
+
+  "RAW_MATERIAL.SHELF_LIFE": "Durée de vie",
+
+  "PUSH": "POUSSER !",
+
+  "MONSTER.NAME": "Nom",
+  "MONSTER.DICE": "Dés d'attaque",
```
Comment (Perfectro) [inline lang/fr.json:129]: A bit long. `Dés` should suffice ?

**Comment by cateland** [comment]
@Perfectro i did no try to boot a campaign with this module, no doubt that i could break the layout with long translation.
I guess it will happen anyway with a swedish translation.
In meantime until i can properly try it i will shortens some.

**Comment by cateland** [comment]
@Perfectro btw thanks for the module ! I will probably use it very soon

**Comment by Perfectro** [comment]
As a side note. I will probably rework a bit all sheets to be larger.
So that should fix it.

**Comment by Perfectro** [comment]
Did some modification on my end.
French translation added on 1.8
Thanks @cateland !

**Pull request closed without merging**